### ISSUE TITLE
Does not build

### ISSUE THREAD

**Issue by georgewfraser** [TypeFox/ls-api#7]
A clean checkout fails with:

```
[ERROR] Plugin org.eclipse.xtend:xtend-maven-plugin:2.10.1-SNAPSHOT or one of its dependencies could not be resolved: Could not find artifact org.eclipse.xtend:xtend-maven-plugin:jar:2.10.1-SNAPSHOT -> [Help 1]
```

If you are using snapshot versions, please configure the snapshot repositories appropriately in pom.xml.

**Comment by svenefftinge** [comment]
@dhuebner could you please fix this?

**Comment by dhuebner** [comment]
@georgewfraser Use mvn -Psnapshots clean install

**Comment by georgewfraser** [comment]
It should not be necessary to know a special command, that I read off a github issue, in order to build the project.

**Comment by spoenemann** [comment]
This has been fixed by converting to a Gradle build (#10).
